cli.js

```diff
@@ -99,6 +99,8 @@ function handleFkillError(processes) {
 
 if (cli.input.length === 0) {
 	init(cli.flags);
+} else if (cli.flags.force || cli.flags.f) {
+	fkill(cli.input, cli.flags);
```

Comment:
Would prefer to instead do this conditional inside the `.catch()` on line 105, so not to create more top-level code-paths.

Could for example do:

```js
const promise = fkill(cli.input, cli.flags);

if (cli.flags.force) {
	promise.catch(() => handleFkillError(cli.input));
}
```